### ISSUE TITLE
Publish itchysats to the AppStore

### ISSUE THREAD

**Issue by holzeis** [itchysats/itchysats#2981]
Sign and publish itchsats app to the Apple AppStore.

see also https://developer.apple.com/macos/distribution/

Dependent on #2980

**Comment by klochowicz** [comment]
I think homebrew is *much* easier target, and is a de-facto standard in mac world. I propose closing in favour of #2985 

note: Integrating signing & notarisation, and the AppStore review process would be a world of pain, but it's possible.

**Comment by bonomat** [comment]
I might not be the target user but is anyone using the Apple AppStore for Mac? I think I've never installed a single app from there.

**Comment by holzeis** [comment]
> I might not be the target user but is anyone using the Apple AppStore for Mac? I think I've never installed a single app from there. 

Same here : 🙈

But consider with the M1 you can run apple iphone apps on your MacBook ;)

**Comment by klochowicz** [comment]
@holzeis after re-reading this ticket I might have misunderstood its original intention - did you mean publishing an *iOS* app to the AppStore?

**Comment by holzeis** [comment]
Initially I was referring to the Mac apple store.

**Comment by klochowicz** [comment]
in this case, I'll close it as not planned for now; homebrew is de-facto standard to install/upgrade apps on macos and there's no real benefit from publishing on the Mac App Store.
